kw/platform/aiohttp/session.py

```diff
@@ -21,12 +21,16 @@ class KiwiClientSession(aiohttp.ClientSession):
         Usage::
 
             from kw.platform.aiohttp import KiwiClientSession
-            async with KiwiClientSession() as c:
-                await c.get('https://kiwi.com')
+
+            async with aiohttp.KiwiClientSession() as client:
```

Comment:
This line will be probably fixed by rebasing (there was a PR about typos already merged)